--- libplacebo-v1.7.0/src/meson.build.orig	2018-12-16 15:08:05.000000000 +0100
+++ libplacebo-v1.7.0/src/meson.build	2019-02-20 17:47:50.271428293 +0100
@@ -24,6 +24,7 @@
 
 # work-arounds for shaderc braindeath
 shaderc_names = [
+    'shaderc',
     'shaderc_shared',
     'shaderc_combined',
 ]
@@ -136,7 +136,7 @@
             ],
   }, {
     'name': 'shaderc',
-    'deps': cc.find_library('shaderc_shared', required: get_option('shaderc')),
+    'deps': shaderc,
     'srcs': 'spirv_shaderc.c',
   }, {
     'name': 'vulkan',
